tests/fast-integration/test/fixtures.js

```diff
@@ -0,0 +1,29 @@
+const deploymentYamlString = `apiVersion: apps/v1
```

Comment:
For me it's a bit too generic, what about renaming it to `httpbinDeployment`?

Reply:
It will be gone in next PR where we rewrite that test to https://github.com/kyma-incubator/local-kyma/blob/main/app-connector-example.sh